glTF-Toolkit/src/GLTFLODUtils.cpp

```diff
@@ -335,14 +337,69 @@ namespace
                 node.name += nodeLodLabel;
                 AddIndexOffset(node.id, nodeOffset);
                 AddIndexOffset(node.meshId, meshOffset);
+                if (!node.skinId.empty())
+                {
+                    AddIndexOffset(node.skinId, skinOffset);
+                }
 
                 for (auto Itr = node.children.begin(); Itr != node.children.end(); Itr++)
                 {
                     AddIndexOffset(*Itr, nodeOffset);
                 }
 
                 gltfLod.nodes.Append(std::move(node));
-            };
+            }
+        }
+
+        {
+            auto skins = lod.skins.Elements();
+            for (auto skin : skins)
+            {
+                // post-fix with lod level indication; 
+                // no functional reason other than making it easier to natively read gltf files with lods
+                skin.name += nodeLodLabel;
+                AddIndexOffset(skin.id, skinOffset);
+                AddIndexOffset(skin.skeletonId, nodeOffset);
+                AddIndexOffset(skin.inverseBindMatricesAccessorId, accessorOffset);
+
+                for (auto Itr = skin.jointIds.begin(); Itr != skin.jointIds.end(); Itr++)

```

Comment:
why not for (auto joint : skin.jointIds) ?

Reply:
Following the node.children pattern above

Reply:
I believe you can fix that too then :)